### PR TITLE
Code Review for the Design & UX

### PR DESCRIPTION
**Pull request by majovanilla**
**This is the first code review request to evaluate our page.**

Considerations:

The HTML validation got an error when using` |` to add multiple fonts on the HTML file. 
Should we change it and add multiple links or should we leave it like that?

### CODE REVIEW

**Review by stratospherique — COMMENTED** [review]
### Pull request under the review by Ahmed Mahfoudh TSE

**Review by stratospherique — CHANGES_REQUESTED** [review]
### Great work team :clap: :clap: . However, some requirements that prevents your project's approval needs to be implemented. check below :arrow_down: 

#### Page structure

##### Footer

* [x] It contains centered links (content is not important).

#### Implementation details

##### Each article

* [x] It uses [`article` tag](https://www.w3schools.com/tags/tag_article.asp).
> wrap your article divs inside an `article` tag

### That's it :relaxed: . Keep up the good work :muscle: . When finsihed, kindly resubmit another code review request :pray: .

**Comment by majovanilla** [comment]
Changed the requirements.

Considerations:

1. The HTML validation shows a warning due to the use of the article tag that has no <h1> because we used it to wrap all the article tags inside it.

2. The HTML validation got the previously mentioned error when using `|` to add multiple fonts on the HTML file.

File: README.md [hunk 1]
```diff
@@ -1 +1,34 @@
...
+Linters
+Stickler
+VSCode
+Atom
+
+## Authors
+
+Microverse students:
+@euqueme
+@majovainilla
```
Comment (samgaco) [inline README.md:34]: Nicely documented readme file :+1: 

Some links to improve the appearance of your readme files. A free course [here](https://www.udacity.com/course/writing-readmes--ud777) and some template for inspiration [here](https://github.com/othneildrew/Best-README-Template).

**Review by samgaco — APPROVED** [review]
#### Hi @majovainilla and @euqueme 

### The project looks great :100: it is approved :1st_place_medal: 

-------------

#### Please address this before merging to the master branch :red_circle: 

* You could encode those `|`elements in the URL for the fonts using the [encoding](https://grox.net/utils/encoding.html) for this character , which would be `%7`.

--------------

#### By the way, well done on documenting the pull request :+1: :clap: hope you keep this up, just another petition :arrow_down: 

* **[For your next code review requests]**  Use meaningful names for your Pull Requests

-------------

* _Summary: Please, correct the error with the `|` character. Once that is done, your project is already approved so you can merge to the master branch and submit the project completion form right away. You could take some time to improve the appearance of the readme file but that is optional._ 

* ###### STATUS OF THE PROJECT: APPROVED :heavy_check_mark:

**Pull request merged**